### PR TITLE
Replace the package downloads with normal package references.

### PR DESCRIPTION
**Pull request by dennisdoomen**

### CODE REVIEW

File: Build/_build.csproj [hunk 1]
```diff
@@ -10,12 +10,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageDownload Include="GitVersion.CommandLine.DotNetCore" Version="[5.0.1]" />
-    <PackageDownload Include="NSpec" Version="[1.0.13]" />
-    <PackageDownload Include="NSpec" Version="[2.0.1]" />
-    <PackageDownload Include="NSpec" Version="[3.1.0]" />
-    <PackageDownload Include="nunit.runners" Version="[2.7.1]" />
-    <PackageDownload Include="xunit.runner.console" Version="[2.4.1]" />
+    <PackageReference Include="GitVersion.CommandLine.DotNetCore" Version="[5.0.1]" />
```
Comment (lg2de) [inline Build/_build.csproj]: Wondering whether "[" and "]" here are really intended or just left over.
I could not found documentation for these characters for `PackageReference`.
Further, your change is inconsistent on this. Is this really your intention?

Comment (eNeRGy164) [reply Build/_build.csproj]: [Package versioning - Version ranges and wildcards](https://docs.microsoft.com/en-us/nuget/concepts/package-versioning#version-ranges-and-wildcards)


Notation | Applied rule | Description
-- | -- | --
[1.0] | x == 1.0 | Exact version match

**Comment by dennisdoomen** [comment]
I'll close it. It doesn't fix the original problem anyway.

**Comment by matkoch** [comment]
What is the original problem?

**Pull request closed without merging**